src/software/ai/hl/stp/play/ball_placement_play_test.py

```diff
@@ -11,10 +11,8 @@
 from proto.ssl_gc_common_pb2 import Team
 from proto.ssl_gc_geometry_pb2 import Vector2
 
-# TODO (#2599): Remove Duration parameter from test
-@pytest.mark.parametrize(
-    "run_enemy_ai,test_duration", [(False, 20)]
-)  # , (True, 20)]) # TODO (#2690): Robot gets stuck in corner of defense area
+
+@pytest.mark.parametrize("run_enemy_ai,test_duration", [[False, 10], (True, 10)])
 def test_two_ai_ball_placement(simulated_test_runner, run_enemy_ai, test_duration):
```

Comment:
That would be a good idea to add- I still can't actually run the test with visualizer on (that weird bug with the stylesheets that we never figured out)

Reply:
We should spend some time trying to figure out that bug. Maybe we could add an arg that removes the stylesheet or something for people with that bug. Since it's pretty important to be able to visualize tests

Reply:
James is working on this test currently, for so this PR I don't think its wise to touch this file, as its getting changed anyway